### PR TITLE
Add radius to unscaled pack layout

### PR DESCRIPTION
**Pull request by makyo**
Note that, as this relies on external library testing (D3), tests for this are not easily reproduced without reproducing an entire test suite.  Thankfully, it only affects layout.  If reviewers feel that they are warrented, however, I can dig into finding a test solution for this that tests the effects of radius on service layout.

Addresses https://bugs.launchpad.net/juju-gui/+bug/1454367

### CODE REVIEW

**Comment by makyo** [comment]
# QA

* `make devel`
* copy [this](https://gist.github.com/makyo/06b4126d4740bb117a76) delta
* open the console for the app
* run `app.env._handleRpcResponse(<paste of the delta>)`
* note the spacing, then reload, or open a new tab to the devel server
* repeat for [this](https://gist.github.com/makyo/6532d5374e7a55f76012#file-delta-json) delta (note, just the first file in the gist)
* the spacing should be the same between the two
* the same experiment repeated on demo.jujucharms.com or develop should show the instance with more services clustering the services closer together.

**Comment by mitechie** [comment]
:+1: on the code :)

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2515/
Test FAILed.

File: app/views/topology/service.js [hunk 1]
```diff
@@ -1220,6 +1220,7 @@ YUI.add('juju-topology-service', function(Y) {
                    .size([width, height])
                    .value(function(d) { return Math.max(d.unit_count, 1); })
                    .padding(300)
```
Comment (hatched) [inline app/views/topology/service.js]: Could you add some comments here as to why each one of these chainings is required? 

Also re-your test, This return a data structure that you can compare against by chance?

**Comment by hatched** [comment]
:+1: on the code with some comments plz

**Comment by hatched** [comment]
QA OK using this delta set https://gist.github.com/makyo/acd3bf73254a87a8942d which doesn't have position annotations.

**Comment by makyo** [comment]
Thanks! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2518/
Test FAILed.

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/juju-gui-merge/883

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2519/
Test FAILed.

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/juju-gui-merge/884

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2521/
Test PASSed.

**Pull request merged**